core/domain/skill_services_test.py

```diff
@@ -730,8 +731,8 @@ def _mock_logging_function(msg, *args):
 
         self.assertEqual(len(observed_log_messages), 1)
         self.assertEqual(
-            observed_log_messages[0], 'AttributeError \'str\' object has no '
-            'attribute \'cmd\' %s invalid_change_list' % self.SKILL_ID)
+            observed_log_messages[0], 'AttributeError \'unicode\' object has no'
```

Comment:
Will this need to be changed back to `str` after we move to Python 3?

Reply:
Yep

Reply:
Then use `self.assertRegexpMatches(log_msg, 'object has no attribute \'cmd\'')` instead since that's what this test really cares about (not whether it was a str or unicode).

Reply:
Done